azure-pipelines.yml

```diff
@@ -90,9 +94,20 @@ jobs:
     workingDirectory: dldt-build
     displayName: 'IE CAPITests'
     continueOnError: false
+#  - script: |
```

Comment:
is it draft pull request?

Reply:
partially, but good enough for next step